### PR TITLE
snapctl: added long help to stop/start/restart command

### PR DESCRIPTION
**Pull request by stolowski**
Added long help to `snapctl stop/start/restart`; this is a followup to the implementation PR that just landed.

### CODE REVIEW

File: overlord/hookstate/ctlcmd/restart.go [hunk 1]
```diff
@@ -27,10 +27,13 @@ import (
 
 var (
 	shortRestartHelp = i18n.G("Restart services")
+	longRestartHelp  = i18n.G(`
+The restart command restarts given services of the snap. If executed from the
```
Comment (pedronis) [inline overlord/hookstate/ctlcmd/restart.go]: the given services of the snap.

File: overlord/hookstate/ctlcmd/stop.go [hunk 2]
```diff
@@ -35,10 +35,13 @@ type stopCommand struct {
 
 var (
 	shortStopHelp = i18n.G("Stop services")
+	longStopHelp  = i18n.G(`
+The stop command stops given services of the snap. If executed from the
```
Comment (pedronis) [inline overlord/hookstate/ctlcmd/stop.go]: the given services of the snap.

File: overlord/hookstate/ctlcmd/start.go [hunk 3]
```diff
@@ -27,10 +27,13 @@ import (
 
 var (
 	shortStartHelp = i18n.G("Start services")
+	longStartHelp  = i18n.G(`
+The start command starts given services of the snap. If executed from the
```
Comment (mvo5) [inline overlord/hookstate/ctlcmd/start.go]: `The start command starts the given ...` (I think there is a missing `the` here).

File: overlord/hookstate/ctlcmd/restart.go [hunk 4]
```diff
@@ -27,18 +27,21 @@ import (
...
+	addCommand("restart", shortRestartHelp, longRestartHelp, func() command { return &restartCommand{} })
 }
 
 type restartCommand struct {
 	baseCommand
 	Positional struct {
 		ServiceNames []string `positional-arg-name:"<service>" required:"yes"`
 	} `positional-args:"yes" required:"yes"`
-	Reload bool `long:"reload"`
+	Reload bool `long:"reload" description:"reload the given services if they support it (see man systemctl for details)"`
```
Comment (pedronis) [inline overlord/hookstate/ctlcmd/restart.go]: the style looking at snap install -h  is for this to be capatilized "Reload..."

**Comment by mvo5** [comment]
Looks good to me, slightly unsure about the systemctl referene maybe @chipaca has an opinion?

**Pull request merged**